bin/src/app.rs

```diff
@@ -101,10 +101,16 @@ impl EthUIApp {
     }
 
     pub fn run(self) {
-        self.app.run(|_, event| {
-            if let tauri::RunEvent::ExitRequested { api, .. } = event {
+        self.app.run(|_, event| match event {
```

Comment:
the first argument (currently `_`) is an `AppHandle`, so I think you can just directly call `utils::all_windows_focus(handle)` down there instead of going through a broadcast

Reply:
tried that but ran into some lifetimes issues with regard to the `AppHandle` 

```error[E0521]: borrowed data escapes outside of closure
   --> bin/src/app.rs:112:17
    |
104 |         self.app.run(|app_handle: &AppHandle, event| match event {
    |                       ----------  - let's call the lifetime of this reference `'1`
    |                       |
    |                       `app_handle` is a reference that is only valid in the closure body
...
112 |                 tokio::spawn(async { all_windows_focus(app_handle).await });
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |                 |
    |                 `app_handle` escapes the closure body here
    |                 argument requires that `'1` must outlive `'static`
```